scripts/gen_rancid.php

```diff
@@ -50,7 +50,7 @@
         if ($devices['disabled']) {
             $status = "down";
         }
-        echo $devices['hostname'] . ':' . $rancid_map[$devices['os']] . ':' . $status . PHP_EOL;
```

Comment:
This will just break compatibility.

You'd need to add a config option so people can specify which version people are running.

Also, as mentioned [in](https://github.com/librenms/librenms/pull/9027), you need to split out your pull requests. docs changes for transports and a rancid change in one won't get merged. You also shouldn't make changes in your master branch. We have docs on using git, have a read.

Reply:
Thx for and idea. I'll close this pull request and make changes there first